AsyncDisplayKit/ASDisplayNode.mm

```diff
@@ -703,6 +704,69 @@ - (instancetype)styledWithBlock:(void (^)(ASLayoutElementStyle *style))styleBloc
 
 #pragma mark - Layout
 
+- (void)invalidateSize
```

Comment:
Just wanted to say, this is a very smart and elegant method.  It's implemented well and accomplishes a very interesting behavior without much code, and is one of the most crucial keys to the great efficiency wins that this diff gives us.

We will think about a final name after this diff lands.  One idea - setNeedsLayoutFromAbove?  The interesting thing about this method is that it doesn't just invalidate, but actively climbs the tree.